go/actions/test/action.yml

```
@@ -0,0 +1,76 @@
+name: Test go extractor
+description: Run build, QL tests and optionally basic code sanity checks (formatting and generation)
+inputs:
+  go-version:
```

Minor: to avoid ambiguity at the use site, perhaps this could include the word "test" somewhere?
```suggestion
  go-test-version:
```

---

great suggestion!